CIP-0062/README.md

```diff
@@ -0,0 +1,530 @@
+---
+CIP: 62
+Title: Cardano dApp-Connector Governance extension
+Category: Wallets
+Status: Proposed
+Authors:
+    - Bruno Martins <bruno.martins@iohk.io>,
+    - Steven Johnson <steven.johnson@iohk.io>,
+    - Ryan Williams <ryan.williams@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/296
+Created: 2021-06-11
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This document describes an interface between webpage/web-based stacks and Cardano wallets. This specification defines the API of the javascript object that needs to be injected into web applications to support governance features.
+
+These definitions extend [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide specific support for vote delegation and vote signing in cardano.
+
+# Motivation
+
+Cardano requires governance capabilities, to support not only the Catalyst
+system which is funded by the Cardano Treasury, but also for other voting
+purposes. To be able to participate in a voting system users associate their
+mainnet staked ADA with a "voting key". This association happens on
+Cardano mainnet via metadata attached to a "registration" transaction.
+
+Registration uses
+[CIP-36 (Catalyst/Voltaire Registration Transaction Metadata Format - Updated)](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036)
+which defines a metadata standard for voter registration, accompanied by a
+defined voting key derivation path. This standard brings a suite of benefits,
+namely:
+
+* vote delegation to either private or public representatives (dReps)
```

Comment:
Should we remove the term dRep from here? since it could be confused with Voltaire dReps and I think explicitly stating that they are Catalyst dReps tailors it too far to Catalyst

Reply:
Is there any reason why Voltaire dreps and catalyst dreps can't use the same registration process?  Given its a CIP standard all its own I am leaning into that.  Voltaire should probably register itself as Purpose 1.